sonar-duplications/src/main/java/net/sourceforge/pmd/cpd/TokenEntry.java

```diff
@@ -27,7 +27,9 @@
 
 /**
  * @since 2.2
+ * @deprecated since 5.5
  */
+@Deprecated
 public class TokenEntry implements Comparable<TokenEntry> {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)